### PR TITLE
add separate databases for multiple accounts

### PR DESCRIPTION
**Pull request by a13sha**
may need to change test files I don't know how to do it correctly
image

### CODE REVIEW

**Comment by ohld** [comment]
Still Travis failed.

File: instabot/bot/bot.py [hunk 1]
```diff
@@ -44,17 +43,20 @@
 from .bot_unlike import (unlike, unlike_comment, unlike_media_comments,
                          unlike_medias, unlike_user)
 from .bot_video import upload_video
+from .bot_files import (set_whitelist, set_blacklist, set_comments, set_followed,
```
Comment (ohld) [inline instabot/bot/bot.py]: codestyle:
```
from .bot_files import (
  set_whitelist, 
  set_blacklist, 
  set_comments, 
  set_followed,
  set_unfollowed, 
  set_skipped, 
  set_friends
)
```

**Comment by lexan73** [comment]
fixed
screen shot 2019-02-25 at 4 36 37 pm

**Review by ohld — DISMISSED** [review]

File: instabot/bot/bot.py (hunk 1 above)
Comment (a13sha) [reply instabot/bot/bot.py]: I got it thanks but this string no longer exists

**Pull request closed without merging**